pycape/cape_encrypt.py

```diff
@@ -0,0 +1,48 @@
+import os
+from typing import Tuple
+
+from cryptography import x509
+from cryptography.hazmat.primitives import hashes
+from cryptography.hazmat.primitives.asymmetric import padding
+from cryptography.hazmat.primitives.asymmetric import rsa
+from cryptography.hazmat.primitives.ciphers import aead
+
+
+def encrypt(message: bytes, key: bytes) -> bytes:
+    # cape key is DEM-encoded RSA key
+    rsa_key = _parse_rsa_key(key)
+    # create ephemeral AES key
+    aes_key = _aes_keygen(32)
+    # encrypt message w/ AES
+    data_ctxt, nonce = _aes_encrypt(message)
+    # encrypt the ephemeral AES key w/ RSA
+    key_ctxt = _rsa_encrypt(aes_key, rsa_key)
+    # concatenate everything into a single bytes obj
+    final_ctxt = key_ctxt + nonce + data_ctxt
```

Comment:
Was looking for this. Wanted to make sure we have the nonce prefixing the aes ciphertext.